portlets/src/main/webapp/vue-app/admin-connectors-settings/components/GamificationAdminConnectorCardList.vue

```diff
@@ -0,0 +1,49 @@
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 - 2023 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-card flat>
+    <div class="text-header-title py-2 py-sm-5 d-flex align-center">{{ $t('gamification.connectors.label.connectors') }}</div>
+    <v-card-text class="text-font-size pb-2 pt-0 pb-sm-5 px-0 d-flex align-center">{{ $t('gamification.connectors.label.valueContributions') }}</v-card-text>
+    <v-card-text class="pa-0">
+      <v-item-group>
+        <v-container class="pa-0">
+          <div v-if="enabledConnectors && enabledConnectors.length" class="ma-0 d-flex flex-wrap">
```

Comment:
To move to parent IMO
```suggestion
          <div class="ma-0 d-flex flex-wrap">
```